stl/inc/type_traits

```diff
@@ -1705,8 +1717,8 @@ struct _Invoke_traits_nonzero {
 };
 
 template <class _Callable, class _Ty1, class... _Types2>
-using _Decltype_invoke_nonzero = decltype(
-    _Invoker1<_Callable, _Ty1>::_Call(_STD declval<_Callable>(), _STD declval<_Ty1>(), _STD declval<_Types2>()...));
+using _Decltype_invoke_nonzero = decltype(_Invoker1<_Callable, _Ty1>::_Call(
+    _STD declval<_Callable>(), _STD declval<_Ty1>(), _STD declval<_Types2>()...));

```

Comment:
It seems there are some strange formatting changes

Reply:
I think I might be using a wrong clang-format version :/

Reply:
Ah, the repo hasn't updated to 16.11 Preview 1 yet so still on clang-format 11.